book/chrome.md

```diff
@@ -509,7 +509,7 @@ tease tabs and browsers apart.
 Here's the plan: the `Browser` class will own the window and canvas
 and all related methods, such as event handling. And it'll also
 contain a list of `Tab` objects and the page chrome. But the web page
```

Comment:
By the way, can I understand that page chrome and browser chrome mean the same thing?
page chrome is not explained while browser chrome is explained in the section below.
(The term 'chrome' first appeared in this chapter.)

Reply:
This is also a good catch, it should be browser chrome. In the interests of time, I fixed it in #1337